tests/console/git.pm

```diff
@@ -22,12 +22,12 @@ use base "consoletest";
 use strict;
 use warnings;
 use testapi;
-use utils;
+use utils qw(zypper_call);
 
 sub run {
-
     my $username = $testapi::username;
     my $email    = "you\@example.com";
+    my ($self)   = shift;
```

Comment:
Not needed?

Reply:
This was outdated when you commented it.